tests/test_paramviewer.py

```diff
@@ -10,14 +10,14 @@ def test_paramviewer_simple_nonbatched(backend):
     view = ParamViewer(
         parshape,
         {'hello': {'slice': slice(0, 2)}, 'world': {'slice': slice(5, 7)}},
-        ['hello', 'world'],
+        ['world', 'hello'],
     )
     par_slice = view.get(pars)
-    assert pyhf.tensorlib.tolist(par_slice[view.slices[0]]) == [1, 2]
+    assert pyhf.tensorlib.tolist(par_slice[slice(2, 4)]) == [1, 2]
 
-    assert pyhf.tensorlib.tolist(par_slice[view.slices[1]]) == [6, 7]
+    assert pyhf.tensorlib.tolist(par_slice[slice(0, 2)]) == [6, 7]
```

Comment:
these two tests don't really do anything that's not covered by the third. I suggest maybe testing something like `par_slice['world'] == [1,2]` but it doesn't seem like that's possible. I suppose that's not the point since `ParamViewer` already filters out the `hello` and `world` for you...

Reply:
I didn't want to remove the tests just b/c I removed the attr, but I agree they're not adding anything

Reply:
ok, i agree. it doesn't necessarily hurt, just extra unneeded code.